ios-target-app/Home/Views/CreateTargetBottomSheetView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CreateTargetBottomSheetView.swift
+//  ios-target-app
+//
+//  Created by Raul Piñeres Carrera on 24/10/22.
+//
+
+import UIKit
+
+protocol BottomSheetDelegate: AnyObject {
+  func createTargetButtonTapped()
+}
+
+final class CreateTargetBottomSheetView: UIView {
+  
+  private lazy var targetImageView = UIImageView(imageResource: "ic_target")
+  
+  private lazy var createTargetButton = UIButton(
+    color: .clear,
+    text: "home_create_target_button_label".localized,
+    textColor: .black,
+    target: self,
+    action: #selector(createNewTargetTapped)
+  )
+  
+  weak var delegate: BottomSheetDelegate?
+  
+  init() {
+    super.init(frame: .zero)
+    
+    addSubviews([
+      targetImageView,
+      createTargetButton
+    ])
+    
+    layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
+    layer.cornerRadius = UI.BottomSheet.cornerRadius
+    layer.shadowRadius = UI.BottomSheet.shadowRadius
+    backgroundColor = .white
```

Comment:
i would encapsulate this in a single func to decouple the code like i.e `configureViews`, also for constrains setup below